### ISSUE TITLE
Condensed and Narrow font

### ISSUE THREAD

**Issue by domoritz** [bBoxType/FiraGO#22]
I see that Fira supports narrow and condensed but Fira Go does not. Are these variants on the roadmap?

**Comment by carrois** [comment]
Yes, on our roadmap already. We are still waiting for a sponsor ;-)

**Comment by domoritz** [comment]
I obviously don't have the means to pay for it but have you considered crowdfunding part of the development or asking users of Fira like Here or the government of New Zealand?

**Comment by phoberus** [comment]
Do you have an update regarding FiraGO condensed? Would love to use it.
